### ISSUE TITLE
bet365 detecting

### ISSUE THREAD

**Issue by mmbertasi** [fysh711426/UndetectedChromeDriver#42]
Hello,

This has been super useful so thank you very much for your efforts.  Very recently bet365.mx or bet365.com etc has begun detecting undetected chromedriver and I think it has something to do with the CDP runtime.  Is there anyway to mask/hide this? If you have a minute to test and have any other thoughts they are most welcome

**Comment by fvzptr** [comment]
I have the same problem too, since this morning I've been trying to debug the problem. The problem started quite recently, before sporadically now it seems to be difficult to overcome the obstacle.
As an example, if it can be useful, at this link --> [Link](https://www.bet365.it/#/AC/B1/C1/D8/E133691742/F3/I0/) 

if i open with a regular chrome i see this.
![regular](https://user-images.githubusercontent.com/43762626/224185019-8fbcd9f9-7a48-48cf-af1f-ec1b2b71780c.png)

If instead I open with undetectedChromeDriver I see this.
![undetechromedriver](https://user-images.githubusercontent.com/43762626/224185018-bb1f8074-11f5-482f-9dd7-63828053426f.png)

Chromedriver and Chrome are updated to the latest release available (111) but the problem is also encountered on previous versions.
I tried to change fingerprint, disable webgl , change screensize etc ....
I also tried automatisms such as mouse movements, clicks, opening new tabs. Nothing works.

Please help us
thank so much

**Comment by fysh711426** [comment]
https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/611
This problem seems to have existed for a long time and has not been resolved.

**Comment by liorskater** [comment]
Hello and thanks for your driver !
bet365 it not working for me since 08-03-2023
They now detecting us

**Comment by fvzptr** [comment]
> [ultrafunkamsterdam/undetected-chromedriver#318](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/318) [ultrafunkamsterdam/undetected-chromedriver#611](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/611) This problem seems to have existed for a long time and has not been resolved.

Okay. But it doesn't seem like the same problem to me. I do not log in. I Go to link match and find element . In a regular chrome work. But with undetectedchromedriver , from yesterday , not anymore. Does not load the odds grid.

**Comment by liorskater** [comment]
![image](https://user-images.githubusercontent.com/10012960/224322170-e53211d2-a3a3-4aca-8e3e-337fc224c766.png)
I tried also the regular chrome driver , and i steel getting the same results.

**"Sorry, there are no markets currently available in this category."**

But If i try this page from regular chrome browser I  get all the data
but from any selenium (undected or ChromDriver()) i cannot see the data, bet365 somehow manage to block us from scrapping data. this is a very big problem for me. 

https://www.bet365.com/#/AS/B16/

if you have any idea I will be happy to hear it,
**Thanks you**

**Comment by stavgan-yep** [comment]
Yes, bet365 also stopped working for me, I noticed that if you open the second tab, then everything works on it. When I call the browser.SwitchTo().Window(browser.WindowHandles[1]) method, it stops working on the second tab(. Does anyone have any ideas?

![image](https://user-images.githubusercontent.com/52219468/224424682-436b5d1a-7150-44ca-8acb-e7e009dac277.png)
![image](https://user-images.githubusercontent.com/52219468/224424716-3c61a786-e5f8-4d82-860d-b5b5fd8318fc.png)

**Comment by liorskater** [comment]
Thanks i now try to play with Js functions, if i copy -> paste -> and press
enter is work,
i will try to mimic this behavior


‫בתאריך יום ו׳, 10 במרץ 2023 ב-22:46 מאת ‪stavgan‬‏ <‪
***@***.***‬‏>:‬

**Comment by natancordeiro** [comment]
Does anyone know if you can use "subprocess.Popen"? With it you can open a chrome passing the parameter "--remote-debugging" to debug remotely. But I don't know how to implement this, I don't know if there's a way to open it on localhost: or something like that, if anyone knows if there's a way to do it this way, I left the idea.

**Comment by Mattesanti10** [comment]
Has anyone found a solution?

**Comment by cosmoras** [comment]
Could you please make sure that all the settings and hints listed on the following article have been followed when trying to reproduce? PLEASE UPDATE IF SOMEBODY resolve it.
